src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -982,9 +982,11 @@ IdleConnectionResiliency getSessionRecovery() {
 
     /** global system ColumnEncryptionKeyStoreProviders */
     static Map<String, SQLServerColumnEncryptionKeyStoreProvider> globalSystemColumnEncryptionKeyStoreProviders = new HashMap<>();
+    static boolean isWindows;
```

Comment:
maybe cleaner to just define isWindows in the same block below

Reply:
Its being used again further down below.

Reply:
Can't define in the static block below, otherwise isWindows can't be used globally and then we can't use it in `SQLServerColumnEncryptionCertificateStoreProvider `. I've just defined it on line 985 instead of completely in the block.